README.md

```diff
@@ -1,144 +1,97 @@
-
-- [`./deploy-container-minimal.sh`](examples/deploy-container-minimal.sh) creates a
-  container defined by [minimal-configuration.nix](examples/minimal-configuration.nix) that
-  doesn't use the [secure-node.nix](modules/presets/secure-node.nix) preset.
-  Also shows how to use nix-bitcoin in an existing NixOS config.\
-  Requires: [Nix](https://nixos.org/), a systemd-based Linux distro and root privileges
-
-Run the examples with option `--interactive` or `-i` to start a shell for interacting with
-the node:
-```bash
-./deploy-qemu-vm.sh -i
-```
-
-#### Tests
-The internal test suite is also useful for exploring features.\
-The following `run-tests.sh` commands leave no traces (outside of `/nix/store`) on
-the host system.
-
-```bash
-git clone https://github.com/fort-nix/nix-bitcoin
-cd nix-bitcoin/test
-
-# Run a Python test shell inside a VM node
-./run-tests.sh debug
-print(succeed("systemctl status bitcoind"))
-
-# Run a node in a container. Requires systemd and root privileges.
-./run-tests.sh container
-c systemctl status bitcoind
+* Make it easy to deploy a reasonably secure Bitcoin node with a usable wallet
+* Allow providing public infrastructure for Bitcoin and higher-layer protocols
+* Be a usable, reasonably secure platform for trustless Bitcoin yield generation
+* Be a reproducible and extensible platform for applications building on Bitcoin
 
-# Explore a single feature
-./run-tests.sh --scenario electrs container
-```
-See [`run-tests.sh`](test/run-tests.sh) for a complete documentation.
-
-Available modules
+Features
 ---
-By default the `configuration.nix` provides:
-* bitcoind with outbound connections through Tor and inbound connections through a hidden service. By default loaded with banlist of spy nodes.
-* [clightning](https://github.com/ElementsProject/lightning) with outbound connections through Tor, not listening
+Default
+* bitcoind with outbound connections through Tor and inbound connections through an onion service. By default loaded with banlist of spy nodes.
+* [clightning](https://github.com/ElementsProject/lightning) with outbound connections through Tor, onion service unannounced
 * includes "nodeinfo" script which prints basic info about the node
-* adds non-root user "operator" which has access to bitcoin-cli and lightning-cli
-
-In `configuration.nix` the user can enable:
-* a clightning hidden service with [plugins](https://github.com/lightningd/plugins)
-* [liquid](https://github.com/elementsproject/elements)
-* [lightning charge](https://github.com/ElementsProject/lightning-charge)
-* [nanopos](https://github.com/ElementsProject/nanopos)
-* an index page using nginx to display node information and link to nanopos
+* adds non-root user "operator" who has access to client tools (ex. bitcoin-cli, lightning-cli)
+
+In `configuration.nix` the user can enable
+* announcing a clightning onion service and [clightning plugins](https://github.com/lightningd/plugins):
```

Comment:
How about linking to the plugin section in usage.md here? Would save some space. Perhaps we can keep some highlights here, like clboss.

Reply:
I like seeing all the c-lightning plugins. We can remove this section if we add more plugins and it takes up too much space.